src/app/fdctl/run/tiles/fd_dedup.c

```diff
@@ -31,7 +31,7 @@ typedef struct {
   ulong * tcache_ring;
   ulong * tcache_map;
 
-  fd_dedup_in_ctx_t in[ 32 ];
+  fd_dedup_in_ctx_t in[ 64UL ];
```

Comment:
Existing bug, but can you add a `FD_TEST( tile->in_cnt<sizeof(ctx->in)/sizeof(ctx->in[0]) )` to line 153 to not overrun this?